api/src/start.ts

```diff
@@ -1,14 +1,18 @@
 import app from './Server';
-import { config } from './config';
-import { logger } from './logger';
+import { config } from '../config';
+import { logger } from '../logger';
 import { getDbClientInstance } from './database/dbClient';
 import { createDatabase } from './database/createDatabase';
 
 export const runServer = async () => {
     // Create database
     await createDatabase();
-    // Database client
+
+    // Database migrations
     const dbClient = getDbClientInstance();
+    logger.info(`Migrating database ${config.database.connection.database}.`);
+    await dbClient.migrate.latest();
+    logger.info(`Migrated to version ${await dbClient.migrate.currentVersion()}.`);
```

Comment:
Doing an `await` in the logger line is strange. the `await` changes code behavior (suspends you code at this point) but generally we assume it's OK to remove, change, or move log statements.